atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1034,6 +1036,25 @@ void WebContents::OnRendererMessageSync(const base::string16& channel,
 }
 
 // static
+bool WebContents::NotifyZoomLevelChanged(
```

Comment:
We can make this an anonymous function instead of WebContent's method, since it is only used in one place and doesn't access WebContent's members.